components/LandingPageContent/MainLandingPage/PaprMemeBanner.tsx

```diff
@@ -0,0 +1,79 @@
+import { Button } from 'components/Button';
+import Image from 'next/image';
+import CatThing from 'public/landing-page-nfts/cat-thing.png';
```

Comment:
someone tell me what the correct names for these NFTs are :laughing:

Reply:
tubby cats